how-to-enable-trace-listener-for-asp-net-core-projects.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Problems in ASP.NET Core applications
+description: How to enable Trace Listener for ASP.NET Core applications when crashing
+type: how-to
+page_title: How to enable Trace Listener for ASP.NET Core projects 
```

Comment:
The KB is about troubleshooting problems in asp.net core apps. It seems strange that the title and description mention only a trace listener.